### ISSUE TITLE
Superior displacement of the humeral head

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11002]
**Preferred term label:**
Superior displacement of the humeral head

**Synonyms**

**Definition (free text, please give PubMed ID)**
AP shoulder radiograph showing distance between the the proximal aspect of the humeral head and inferior aspect of the acromion of <7mm; decreased acromiohumeral distance (AHD); high-riding shoulder; proximal subluxation of the humeral head

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Rotator cuff tear arthropathy, Rotator cuff tear, Shoulder external impingement syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001171
- label: Superior displacement of the humeral head
- parents: [Abnormal humeral head morphology (HP:0003887)](https://hpo.jax.org/app/browse/term/HP:0003887)
- pmid: PMID:8816337
- def: An anterior posterior (AP) shoulder radiograph shows distance between the the proximal aspect of the humeral head and inferior aspect of the acromion of less than 7 mm.
- comment: Conditions characterized by this finding include rotator cuff tear arthropathy, rotator cuff tear, and shoulder external impingement syndrome.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11002
- synonyms: Proximal subluxation of the humeral head(NONE)|Decreased acromiohumeral distance(NONE)|High-riding shoulder(NONE)
